src/Middleware/UserPageDecorationMiddleware.php

```diff
@@ -135,7 +135,7 @@ function ($a, $b) {
 
         $tpl_data['subtest'] = $request->getAttribute("pageclass")->page ?? null;
 
-        $page = $request->getAttribute("pageclass");
+        $page = $request->getAttribute("pageclass") ?? '';
```

Comment:
The method_exists() line 139 doesn't accept null and needs an empty string for $page.

Reply:
This causes other phan errors, I sent a PR to your branch to fix them